src/extension_commands.ml

```diff
@@ -158,6 +158,48 @@ let _open_ocamllsp_output_pane, _open_ocaml_platform_ext_pane, _open_ocaml_comma
       (handler Output.command_output_channel) )
 ;;
 
+let decorationType =
```

Comment:
This name should describe what does this decoration looks like and/or what are correct use cases for it such as `highlighting_decoration` or `quick_peek_highlighting_decoration`.